etc/crontab.xml

```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/crontab.xsd">
-    <group id="yotpo">
+    <group id="yotpo_core_orders_sync">
+        <job name="yotpo_cron_core_orders_sync" instance="Yotpo\Core\Model\Sync\Orders\Cron\OrdersSync" method="processOrders">
+            <config_path>crontab/yotpo_core_orders_sync/jobs/yotpo_cron_core_orders_sync/schedule/cron_expr</config_path>
+        </job>
+    </group>
+    <group id="yotpo_core_catalog_sync">
         <job name="yotpo_cron_core_products_sync" instance="Yotpo\Core\Model\Sync\Catalog\Cron\CatalogSync" method="execute">
-            <config_path>crontab/yotpo/jobs/yotpo_cron_core_products_sync/schedule/cron_expr</config_path>
+            <config_path>crontab/yotpo_core_catalog_sync/jobs/yotpo_cron_core_products_sync/schedule/cron_expr</config_path>
         </job>
         <job name="yotpo_cron_core_category_sync" instance="Yotpo\Core\Model\Sync\Catalog\Cron\CategorySync" method="execute">
-            <config_path>crontab/yotpo/jobs/yotpo_cron_core_category_sync/schedule/cron_expr</config_path>
-        </job>
-        <job name="yotpo_cron_core_orders_sync" instance="Yotpo\Core\Model\Sync\Orders\Cron\OrdersSync" method="processOrders">
-            <config_path>crontab/yotpo/jobs/yotpo_cron_core_orders_sync/schedule/cron_expr</config_path>
+            <config_path>crontab/yotpo_core_catalog_sync/jobs/yotpo_cron_core_category_sync/schedule/cron_expr</config_path>
         </job>
+    </group>
+    <group id="default">
```

Comment:
As this is a cronjob which is running only once a day, I though to make it as part of default magento cron group. Otherwise if we create it as a separate cron group, it will instantiate a separate PHP process for that, which is actually an unwanted use of resource. Please let me know if we need to change it back to "yotpo"